eslint.config.mjs

```diff
@@ -0,0 +1,64 @@
+import { fixupConfigRules } from "@eslint/compat";
```

Comment:
I think we don't need `compat` or `FlatCompat` anymore, as all the plugins/configs that we are using are now compatible with v9. Can you check again?